spec/lib/diaspora_federation/validators/location_validator_spec.rb

```diff
@@ -0,0 +1,23 @@
+module DiasporaFederation
+  describe Validators::LocationValidator do
+    it "validates a well-formed instance" do
+      l = OpenStruct.new(FactoryGirl.attributes_for(:location_entity))
+      v = Validators::LocationValidator.new(l)
+      expect(v).to be_valid
+      expect(v.errors).to be_empty
+    end
+
+    context "#lat and #lng" do
+      %i(lat lng).each do |prop|
+        it "must not be empty" do
+          l = OpenStruct.new(FactoryGirl.attributes_for(:location_entity))
+          l.public_send("#{prop}=", "")
```

Comment:
can you do this with `.merge(data)`, like I did this in the oder validator specs?